platform-sdk/sdk/settings.txt

```diff
@@ -11,49 +11,56 @@
                                           #    Gossip / Networking    #
                                           #############################
 
-chatter.useChatter,               false

```

Comment:
these should probably be reverted

Reply:
This change was intentional. 

- I alphabatized the settings names within each category. 
- I also fixed the indentation of the right column. If we want to keep these aligned, we need to shift the column over whenever we add a setting with a longer name than the others. If this is undesirable, we can just shift everything so that there is a single " " between the setting and the value. Might not look as nice, but we won't have to change it in the future again.